middleware_test.go

```diff
@@ -22,14 +23,21 @@ import (
 	"github.com/acronis/go-authkit/jwt"
 )
 
+const (
+	errDomain       = "TestDomain"
```

Comment:
[minor] How about renaming `errDomain` to `testErrDomain` to be consistent with `testBearerToken`?